packages/ethers-5/src/utils/helpers.ts

```diff
@@ -0,0 +1,76 @@
+import type { CaipNetwork } from '@web3modal/scaffold'
+import { ConstantsUtil, PresetsUtil } from '@web3modal/utils'
+import {
+  NetworkBlockExplorerUrls,
+  NetworkNames,
+  NetworkRPCUrls,
+  networkCurrenySymbols
+} from './presets.js'
+import type { ethers } from 'ethers'
+import type EthereumProvider from 'node_modules/@walletconnect/ethereum-provider/dist/types/EthereumProvider.js'
+
+export function getCaipDefaultChain(chain?: number) {
```

Comment:
Some of these utils look like they are not ethers specific (i.e. don't rely on using ethers or wagmi package), so they can be moved to shared utils? At least CAIP ones?

Reply:
We need to use specific presets that are `ethers` related, also `wagmi` takes in specific `wagmi` chain. So I think it is better to keep it like this.